src/Reference/Task8.java

```diff
@@ -0,0 +1,44 @@
+package Reference;
+
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Task8 {
+
+    public static void main(String[] args) {
+
+        //forexample
+//        String string = "ae 02 eo oo ao aa aeo sdf dkfm fkfkfk og Hello there this is a simple text for this awesome code 1234567";
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter a fraze: ");
+        String string = reader.nextLine();
+
+        String[] split = string.split(" ");
+        Pattern pattern = Pattern.compile("[aeo]");
+
+        for (int j = 0; j < split.length; j++){
+
+            Matcher matcher = pattern.matcher(split[j]);
+            if (matcher.find()){
+                System.out.println(true + " " + split[j]);
+            }
+            else {
+                System.out.println(false + " " + split[j]);
+            }
+        }
+
+        for (int i = 0; i < split.length; i++){
+            if (split[i].length() > 6){
```

Comment:
could you add a solution using a regex pattern ?

Reply:
Pattern pattern = Pattern.compile("\\S[a-zA-Z0-9]{6,}");
        Matcher matcher = pattern1.matcher(string);
        while (matcher.find()) {
            System.out.println(matcher.group());
        }